### PR TITLE
First commit for spinner component with fixed size

### PR DESCRIPTION
**Pull request by jbuican14**
This is an initial commit for loading icon or spinning icon as we know. It will be used when data is being fetched or page is being loaded. Style will be added more for responsive design once the designer gives a mock up for the spinner/loading icon in material UI

### CODE REVIEW

File: front-end/src/components/Spinner.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const useStyles = makeStyles(() => ({
+  spinner: {
+    margin: 'auto',
+    display: 'block',
+    width: '120px',
+    height: '120px',
+  },
+}));
+const Spinner = () => {
```
Comment (kitravee) [inline front-end/src/components/Spinner.jsx:13]: `const Spinner = ({ className }) => {`

className Prop to receive over Styling from Parent

File: front-end/src/components/Spinner.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
```
Comment (kitravee) [inline front-end/src/components/Spinner.jsx:4]: import clsx from 'clsx';

clsx = function to adding the style in the same Component

File: front-end/src/components/Spinner.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  spinner: {
+    margin: 'auto',
+    display: 'block',
+    width: '120px',
+    height: '120px',
+  },
+}));
+const Spinner = () => {
+  const classes = useStyles();
+  return <CircularProgress disableShrink className={classes.spinner} />;
```
Comment (kitravee) [inline front-end/src/components/Spinner.jsx:15]: className={clsx(classes.root, className)}

**Review by kitravee — CHANGES_REQUESTED** [review]

File: front-end/src/components/Spinner.jsx (hunk 1 above)
Comment (jbuican14) [reply front-end/src/components/Spinner.jsx]: added ;)

File: front-end/src/components/Spinner.jsx (hunk 3 above)
Comment (jbuican14) [reply front-end/src/components/Spinner.jsx]: added the call for this --Done

**Comment by jbuican14** [comment]
@kitravee all new code updated for clsx done.

**Review by kitravee — APPROVED** [review]

**Pull request merged**